app/workers/hit_fox_feed_worker.rb

```diff
@@ -0,0 +1,14 @@
+require 'hit_fox_feed'
+
+class HitFoxFeedWorker
+  include Sidekiq::Worker
+  include Sidetiq::Schedulable
+
+  recurrence { minutely.second_of_minute(0, 5, 10, 15, 20, 25) }
+
+  def perform
+    feed = HitFoxFeed.new(ENV['GITHUB_USER'],'HitFox',ENV['GITHUB'])
+
+    File.write('app/views/hit_fox_feed/events.json', feed.events)
```

Comment:
Why are you putting it in a file on the server instead of caching it? Rails cache or redis should suffice. Also this logic should be in the hit_fox_feed class.